planning/behavior_path_planner/src/scene_module/pull_out/pull_out_module.cpp

```diff
@@ -176,23 +178,33 @@ BehaviorModuleOutput PullOutModule::plan()
   output.turn_signal_info =
     calcTurnSignalInfo(status_.pull_out_path.start_pose, status_.pull_out_path.end_pose);
 
+  double start_distance = 0.0;
   if (status_.back_finished) {
-    const double start_distance = motion_utils::calcSignedArcLength(
+    start_distance = motion_utils::calcSignedArcLength(
       path.points, planner_data_->self_pose->pose.position,
       status_.pull_out_path.start_pose.position);
-    const double finish_distance = motion_utils::calcSignedArcLength(
-      path.points, planner_data_->self_pose->pose.position,
-      status_.pull_out_path.end_pose.position);
-    updateRTCStatus(start_distance, finish_distance);
-  } else {
-    const double distance = motion_utils::calcSignedArcLength(
-      path.points, planner_data_->self_pose->pose.position,
-      status_.pull_out_path.start_pose.position);
-    updateRTCStatus(0.0, distance);
   }
 
+  const double finish_distance = motion_utils::calcSignedArcLength(
+    path.points, planner_data_->self_pose->pose.position,
+    status_.pull_out_path.start_pose.position);
+  updateRTCStatus(start_distance, finish_distance);
+
   setDebugData();
 
+  uint16_t direction;
+  if (output.turn_signal_info.turn_signal.command == TurnIndicatorsCommand::ENABLE_LEFT) {
+    direction = SteeringFactor::LEFT;
+  } else {
+    direction = SteeringFactor::RIGHT;
```

Comment:
It' OK becuase pull_out can be done to the right road lane, but It will be `RIGHT` even if `DISABLE` .

Reply:
I add STRAIGHT in this commit [da2db2d](https://github.com/autowarefoundation/autoware.universe/pull/1706/commits/da2db2d0f9050bd975ad170a0e7f98747b17fb97)

Reply:
looks good, thaks